grafana-plugin/src/pages/integration/OutgoingTab/OutgoingWebhookFormFields.tsx

```diff
@@ -134,6 +113,52 @@ export const OutgoingWebhookFormFields: FC<OutgoingWebhookFormFieldsProps> = ({
           </Field>
         )}
       />
+      <Controller
+        control={control}
+        name="url"
```

Comment:
shouldn't this field have any validation rules whatsoever?

Reply:
It can be templated so not really sure if we should do here any frontend validation that's why I removed it for now